spotify_test.go

```diff
@@ -15,7 +16,7 @@ func testClient(code int, body io.Reader, validators ...func(*http.Request)) (*C
 			v(r)
 		}
 		w.WriteHeader(code)
-		io.Copy(w, body)
+		_, _ = io.Copy(w, body)
```

Comment:
Is a linter complaining about not checking the return values here?  I see little value in adding `_, _`.

Reply:
Unfortunately yes, the linter doesn't like ignored error values. I could break from the suggested lint, but I tend to try to avoid this and stick to defaults where possible.

Reply:
Yeah, so let's just check the error instead of `_` it out.